docs/source/first-steps.rst

```diff
@@ -0,0 +1,32 @@
+First Steps
+===========
+
+Work Laptop
+-----------
+Check your personal email for login instructions from IT.
+
+Work email
+----------
+Look for an email asking you to change your password that mentions “the Center
+for Reimagining Learning”. Submit a new password, and sign into your now
+accessible account.
+
+Jira/Confluence
+---------------
+Set up an account on Jira/Confluence via Google using your work email address.
+Verify your email address using the link sent from Jira. Use the grid of nine
+dots in the upper left corner to switch between Jira and Confluence.
+
+discuss.openedx.org
+-------------------
+Set up an account on discuss.openedx.org by selecting the Google Account option
+using the Sign Up button in the upper right. Use a tcril.org email address or
+SSO.
+
+GitHub
+------
+Using your personal account for work now? Better change the email too. You will
+be getting many notifications a day and you’ll want them all in one place. Use
+the settings under email and notifications in Github to add your email address
+and change where the notifications go. Be sure to enable 2 Factor 
+Authorization.
```

Comment:
s/Authorization/Authentication/

Reply:
I've added SSO to the JIRA instructions and replace Authorization with Authentication